### DISCUSSION TITLE
Using estimators from `sklearn.feature_selection` on time series

### DISCUSSION THREAD

**Discussion by hliebert** [alan-turing-institute/sktime#4967] [Q&A] [answered]
Is there currently a way to access `SelectFromModel`, `SelectKBest` and other functionality from `sklearn.feature_selection` ?

I was trying to use `SelectFromModel(LassoCV())` in a pipeline, but `TabularToSeriesAdaptor` passes only `X`, not `y`. `FeatureSelection` on the other hand requires the sklearn model to have a `feature_importances_` attribute.

**Comment by fkiraly** [comment, marked as answer]
I think that wasn't previously possible.

I've extended the `TabularToSeriesAdaptor` so it now should be: https://github.com/sktime/sktime/pull/4978

Would appreciate testing, code review and/or feedback!

**Reply by hliebert** [reply]
This is great, thanks! Will test next week.
